src/menu/views/FilterMenu.js

```diff
@@ -25,6 +25,22 @@ const FilterMenu = MenuBuilder.extend({
       return this.g.columns.set("hidden", hidden);
     });
 
+    this.addNode("Hide non-polymorphic sites",(e) => {
+      var threshold = prompt("Enter identity threshold (in percent)", 100);
+      threshold = threshold / 100;
```

Comment:
it's very common to either check, or [`clamp`](https://lodash.com/docs#clamp) user input. It could be as easy as this:

```js
import {clamp} from "lodash"; // header
...
threshold = clamp(threshold / 100, 0, 1);
```